core/state_processor_rip7560.go

```diff
@@ -211,8 +176,19 @@ func ApplyRip7560ValidationPhases(chainConfig *params.ChainConfig, bc ChainConte
 		GasPrice: gasPrice,
 	}
 	evm := vm.NewEVM(blockContext, txContext, statedb, chainConfig, cfg)
```

Comment:
need to (shallow) copy cfg, and then copy the Tracer struct inside it - and modify that.

Reply:
Done - copying the Tracer object